hat/conf/application.conf

```diff
@@ -307,6 +307,15 @@ she {
       endpoint = "insights/common-locations"
       experimental = true
     }
+    {
+      id = "covid19-score"
+      version = "1.0.0"
+      baseUrl = "covid19-score-dev"
+      baseUrl = ${?COVID19_SCORE_URL}
```

Comment:
Please change environment variable to `DROPS_SHE_BASE_URL`. This is the only one available in the current build environment.